Sources/BottomSheetTransitioningDelegate.swift

```diff
@@ -5,10 +5,14 @@
 import UIKit
 
 public final class BottomSheetTransitioningDelegate: NSObject {
-    private let contentHeights: [CGFloat]
+    private var contentHeights: [CGFloat]
     private let startTargetIndex: Int
     private let useSafeAreaInsets: Bool
-    private var presentationController: BottomSheetPresentationController?
+    private var weakPresentationController: WeakRef<BottomSheetPresentationController>?
```

Comment:
It didn't work to use a `weak var`?

Reply:
No, because when we create `BottomSheetPresentationController` on https://github.com/finn-no/BottomSheet/pull/17/files#diff-7e2615b74729ca09fb8e248a00fcde62R45 and set it to a weak var, it will be immediately deallocated. I tried multiple things, only this approach with struct solved the problem.

Reply:
I see 👍